python/pyfunc-server/docker/Dockerfile

```diff
@@ -24,7 +24,7 @@ WORKDIR /pyfunc-server
 RUN if [ ! -z "$GOOGLE_APPLICATION_CREDENTIALS" ]; then gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}; fi
 RUN gsutil cp -r ${MODEL_URL} .
 RUN /bin/bash -c ". activate merlin-model && \
-    sed -i 's/mlflow$/mlflow==1.23.0/' model/conda.yaml && \
+    sed -i 's/mlflow$/mlflow==1.26.1/' model/conda.yaml && \
```

Comment:
Any particular reason to not use the same sed args as we have for batch predictor above (`sed -i 's/\bmlflow[^[:space:]]*/mlflow==1.26.1/'`)?

Reply:
Yes, it should have been the same.